idaes/models/properties/modular_properties/state_definitions/FcTP.py

```diff
@@ -490,6 +494,56 @@ def calculate_scaling_factors(b):
 do_not_initialize = []
 
 
+class FcTPScaler(FTPxScaler):
+    """
+    Scaler for constraints associated with FcTP state variables
+    """
+
+    # Safe to inherit variable_scaling_routine from FTPx
```

Comment:
Standarize this comments through state definitions. FpcTP does not include "safe".